docs/r_walkthrough.rst

```diff
@@ -57,36 +57,28 @@ Note: Everything we are going to do there can be done in Rstudio as
 well, if the rOMERO-gateway package and its dependencies are available
 on your system.
 
-rOMERO-gateway, rJava and R version 3.5.3 have been installed in a Docker image. For more details, see
+See :doc:`setup`. 
 
--  https://github.com/ome/training-notebooks/blob/master/Dockerfile
-
--  https://github.com/ome/rOMERO-gateway
-
--  https://cran.r-project.org/web/packages/rJava/index.html
-
-**Resources:**
---------------
+Resources
+---------
 
 We will use data from the Image Data Resource (IDR).
 
--  IDR data \ https://idr.openmicroscopy.org/webclient/?show=project-51
+-  IDR data `idr0021 <https://idr.openmicroscopy.org/webclient/?show=project-51>`_.
 
--  Notebook \ https://github.com/ome/training-notebooks/blob/master/R/idr0021%20-%20Segmentation.ipynb
+-  Notebook :file:`notebooks/idr0021_Segmentation.ipynb`.
```

Comment:
this is not clickable and downloadable for me. Builds just as a simple text.
![Screen Shot 2020-07-09 at 17 52 49](https://user-images.githubusercontent.com/2478303/87068129-09066980-c20d-11ea-9931-91e2cbe32f75.png)

Reply:
it is a simple text.

Reply:
Aha, okay.

Reply:
After second though, I will link to the notebook in mybinder.